projects/fal/src/fal/auth/auth0.py

```diff
@@ -180,12 +181,11 @@ def validate_id_token(token: str):
 
 
 def verify_access_token_expiration(token: str):
-    from datetime import timedelta
-
     from jwt import decode
 
+    leeway = 60 * 30 * 60  # 30 minutes
     decode(
         token,
-        leeway=timedelta(minutes=-30),  # Mark as expired some time before it expires
+        leeway=-leeway,  # negative to consider expired before actual expiration
```

Comment:
This is an access token, which is what we send to the API.

I want to make sure that if I start a cli operation, it will do all the requests it needs to without suddenly getting an expired token mid-way through the command; since some commands maye make several requests.

So in case we have a token that is "close" to being expired (we set it at 30 mins here) I just consider it expired right away so it can be refreshed and use a clean one instead.